src/styles/box.scss

```diff
@@ -72,18 +72,25 @@
 
 	&__info {
 		margin-bottom: 4px;
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
 
 		&-value {
 			font-size: 11px;
 			line-height: 15px;
 			font-weight: 600;
-			color: $primaryTextColor;
-		}
-
-		&-name {
-			font-size: 10px;
-			line-height: 14px;
-			color: $secondaryTextColor;
+			color: #fff;
+			border-radius: 9px;
+			&.padded {
+				padding: 0 4px 2px;
+			}
+			&.image-name {
```

Comment:
As i see image name should contain all remaining space in a row and only then trimmed. Also doesn't have any background-color.